packages/validation/src/validators/LionWebSyntaxValidator.ts

```diff
@@ -62,7 +62,7 @@ export class LionWebSyntaxValidator {
     /**
      * Check whether `obj` is a JSON object that conforms to the serialization syntax of LionCore.
      * All errors found will be pushed into the `errors` array, if its length is not 0, the check has failed.
-     * @param obj
+     * @param obj 
```

Comment:
Ooh, you're **so** triggering my OCD! :D

Reply:
?